packages/terra-clinical-item-view/src/ItemView.scss

```diff
@@ -27,13 +27,16 @@
     max-width: 60px;
   }
 
+  // Disable selector-class-pattern. Classes defined via assessoryAlignment prop.
+  /* stylelint-disable selector-class-pattern  */
```

Comment:
Couldn't we just update how the class is set in the component so we can follow the proper naming convention? So switch `{ [accessory-${accessoryAlignment}]: accesoryAlignment }` to 
`
{
    accessory-align-top: accessoryAlignment === 'alignTop',
    accessory-align-center: accessoryAlignment === 'alignCenter'
}`

Reply:
Sure thing.